cli/types/cypress.d.ts

```diff
@@ -2720,7 +2720,7 @@ declare namespace Cypress {
     /**
      * Hosts mappings to IP addresses.
      */
-    hosts: null | string[]
+    hosts: string[]
```

Comment:
Is this removed for convenience or because it's actually incorrect?

Reply:
I'll revert this and update the functions that receive `config.hosts` in the server to reflect that hosts can be `null`.

Reply:
https://github.com/cypress-io/cypress/pull/17200/commits/d64f9e483517605b7353b1ba436e0ba7f4ccdc2d